tests/src/Functional/ParagraphsTest.php

```diff
@@ -140,7 +140,83 @@ public function testAccordionParagraph(): void {
+    $this->assertSession()->fieldExists('oe_bt_paragraphs[0][subform][field_oe_paragraphs][0][subform][field_oe_title][0][value]');
+    $this->assertSession()->fieldExists('oe_bt_paragraphs[0][subform][field_oe_paragraphs][0][subform][field_oe_subtitle][0][value]');
+    $this->assertSession()->fieldExists('oe_bt_paragraphs[0][subform][field_oe_paragraphs][0][subform][field_oe_plain_text_long][0][value]');
+
+    $values = [
+      'title[0][value]' => 'Test Fact and figures node title',
+      'oe_bt_paragraphs[0][subform][field_oe_title][0][value]' => 'Fact and figures block',
+      'oe_bt_paragraphs[0][subform][field_oe_link][0][uri]' => 'https://www.google.com',
+      'oe_bt_paragraphs[0][subform][field_oe_link][0][title]' => 'Read more',
+      'oe_bt_paragraphs[0][subform][oe_bt_layout][0][value]' => 2,
+      'oe_bt_paragraphs[0][subform][field_oe_paragraphs][0][subform][field_oe_title][0][value]' => "1529 JIRA Ticket",
+      'oe_bt_paragraphs[0][subform][field_oe_paragraphs][0][subform][field_oe_subtitle][0][value]' => "Jira Tickets",
+      'oe_bt_paragraphs[0][subform][field_oe_paragraphs][0][subform][field_oe_plain_text_long][0][value]' => "Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.",
+    ];
+
+    $this->submitForm($values, 'Save');
+    $this->drupalGet('/node/1');
+
+    // Assert paragraph values are displayed correctly.
+    $this->assertSession()->pageTextContains('Fact and figures block');
+    $this->assertSession()->pageTextContains('Read more');
+    $this->assertSession()->pageTextContains('1529 JIRA Ticket');
+    $this->assertSession()->pageTextContains('Jira Tickets');
+    $this->assertSession()->pageTextContains('Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.');
+  }
+
+  /**
+   * Test icon options event subscriber.
+   */
+  public function testIconOptionsEventsubscriber(): void {
+    $this->drupalGet('/node/add/paragraphs_test');
+    $page = $this->getSession()->getPage();
+    $page->pressButton('Add Fact');
+
+    $this->assertSession()->fieldExists('oe_bt_paragraphs[0][subform][field_oe_icon]');
+    $allowed_values = [
+      '_none',
+      'arrow-down',
+      'box-arrow-up',
+      'arrow-up',
+      'book',
+      'camera',
+      'check',
+      'download',
+      'currency-euro',
+      'facebook',
+      'file',
+      'image',
+      'info',
+      'linkedin',
+      'files',
+      'rss',
+      'search',
+      'share',
+      'twitter',
+      'person-video',
+    ];
+    foreach ($allowed_values as $allowed_value) {
+      $this->assertSession()->elementsCount('css', 'option[value="' . $allowed_value . '"]', 1);
+    }
+    $this->assertSession()->elementsCount('css', 'select#edit-oe-bt-paragraphs-0-subform-field-oe-icon option', 20);
+
+  }
+
+  /**
+   * Adds a field to a paragraph.
```

Comment:
What happened to this doc comment? It used to be "Test Description list paragraphs form." which seems more correct than the new doc comment.